src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,72 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that is happening at a date, and its duration.
+ */
+public class Event extends Task {
+    protected LocalDate at;
```

Comment:
This variable name could perhaps be made more descriptive, especially considering the fact that when specifying a date, the grammatically correct preposition is "on", not "at".
"On the first of August," not "at the first of August."